src/features/walletUnlocker/OnboardingPairing.tsx

```diff
@@ -80,24 +75,16 @@ export const OnboardingPairing = (): JSX.Element => {
         visible={isDownloadCertModalVisible}
         onOk={() => {
           try {
-            const { host, cert, macaroon } = extractHostCertMacaroon(form.getFieldValue('tdexdConnectUrl'));
-            if (host) {
-              dispatch(setTdexDaemonBaseUrl('https://' + host));
-            } else {
-              throw new Error('Tdexd Connect URL is not valid');
-            }
+            const { cert, macaroon } = extractHostCertMacaroon(form.getFieldValue('tdexdConnectUrl'));
```

Comment:
Please rename `extractHostCertMacaroon` since we don't extract `host` anymore.

Reply:
I think the name of the function is correct. It does extract host, cert, and macaroon from a connect URL. We just don't use the host so we ignore it. No need to rename IMO.